tests/ludwig/modules/test_fully_connected_modules.py

```diff
@@ -36,9 +36,26 @@ def test_fc_layer(
     ],
 )
 def test_fc_stack(first_layer_input_size: Optional[int], layers: Optional[List], num_layers: Optional[int]):
-    if first_layer_input_size is None:
-        first_layer_input_size = layers[0]["input_size"]
     fc_stack = FCStack(first_layer_input_size=first_layer_input_size, layers=layers, num_layers=num_layers).to(DEVICE)
     input_tensor = torch.randn(BATCH_SIZE, first_layer_input_size, device=DEVICE)
     output_tensor = fc_stack(input_tensor)
     assert output_tensor.shape[1:] == fc_stack.output_shape
+
+
+def test_fc_stack_passthrough():
+    first_layer_input_size = 10
+    layers = None
+    num_layers = 0
+    output_size = 15
+    fc_stack = FCStack(
+        first_layer_input_size=first_layer_input_size,
+        layers=layers,
+        num_layers=num_layers,
+        default_output_size=output_size,
+    ).to(DEVICE)
+    input_tensor = torch.randn(BATCH_SIZE, first_layer_input_size, device=DEVICE)
+
+    output_tensor = fc_stack(input_tensor)
+
+    assert list(output_tensor.shape[1:]) == [first_layer_input_size]
```

Comment:
nit: if this is meant to test if the FC stack is a no-op, it might be worth doing an `assert np.isclose` between the inputs and outputs to validate that nothing is happening to the values themselves.

Reply:
Thanks for the suggestion! Added.